MANIFEST.in

```diff
@@ -0,0 +1,8 @@
+include *.txt
+include README.rst
```

Comment:
Ah, I missed the `*.txt` line.  Sorry.

I think I included the `MANIFEST.in` for Traits so that anyone who downloaded the tarball from PyPI could use that to create a valid source distribution.